cmds/core/io/cmos.go

```diff
@@ -26,34 +26,36 @@ io (rtcw index value)... # write value to RTC register index [0-13]
 	addCmd(writeCmds, "rtcw", &cmd{rtcWrite, 7, 8})
 }
 
+var c = cmos.GetCMOS()
+
 func cmosRead(reg int64, data memio.UintN) error {
```

Comment:
I think these ought to be rewritten: 
func (c*whatever) cmosRead(...) error {

That global variable scares me.  I realize, however, that, e.g., the flags package does this sort of thing, so maybe I'm worrying too much? But it seems to me it would be cleaner.

Reply:
As long as I don't use the global variable for tests, I wont create any race conditions. :-)